ApplyToBecomeInternal/ApplyToBecomeInternal/Models/ApplicationForm/Sections/AboutConversionSection.cs

```diff
@@ -21,37 +21,57 @@ public AboutConversionSection(ApplyingSchool application) : base("About the conv
 			new[]
 			{
 				new FormField("The name of the school", application.SchoolName)
-			};
-		private IEnumerable<FormField> GenerateContactFields(ApplyingSchool application) =>
-			new[]
+			};	
+		private IEnumerable<FormField> GenerateContactFields(ApplyingSchool application)
+		{
+			var formFields = new List<FormField>();
+			formFields.Add(new FormField("Name of headteacher", application.SchoolConversionContactHeadName));
+			formFields.Add(new FormField("Headteacher's email address", application.SchoolConversionContactHeadEmail));
+			formFields.Add(new FormField("Headteacher's telephone number", application.SchoolConversionContactHeadTel));
+			formFields.Add(new FormField("Name of the chair of the Governing Body", application.SchoolConversionContactChairName));
+			formFields.Add(new FormField("Chair's email address", application.SchoolConversionContactChairEmail));
+			formFields.Add(new FormField("Chair's phone number", application.SchoolConversionContactChairTel));
+			formFields.Add(new FormField("Approver's name", application.SchoolConversionApproverContactName));			
+			formFields.Add(new FormField("Approver's email address", application.SchoolConversionApproverContactEmail));
+			if (application.SchoolConversionContactRole.ToUpper()=="SOMEONE")
```

Comment:
Hi. Because I've been working on the pipeline yesterday and today I know that this field will actually say "Other" and not "Someone else". Would you change the "SOMEONE" to "OTHER" please? Sorry about that.

Reply:
sure thanks